core/src/com/unciv/ui/tilegroups/TileSetStrings.kt

```diff
@@ -51,7 +51,7 @@ class TileSetStrings {
     val tag = "-"
     fun getTile(baseTerrain: String) = getString(tilesLocation, baseTerrain)
     fun getBaseTerrainOverlay(baseTerrain: String) = getString(tileSetLocation, baseTerrain, overlay)
-    fun getTerrainFeatureOverlay(terrainFeature: String) = getString(tileSetLocation, terrainFeature, overlay)
+    fun getTerrainFeatureOverlay(terrainFeatures: Collection<String>) = getString(tileSetLocation, *terrainFeatures.toTypedArray(), overlay)
```

Comment:
If you plan to have the items + delimited, you'll need to add an extra + between every two items

Reply:
done